helm_deploy/values-dev.yaml

```diff
@@ -22,9 +22,7 @@ generic-service:
     API_BASE_URL_ADJUDICATIONS: https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_ALERTS: https://alerts-api-dev.hmpps.service.justice.gov.uk
     API_BASE_URL_COURT_SENTENCING: https://remand-and-sentencing-api-dev.hmpps.service.justice.gov.uk
-    # currently set to self for a mock API until real API is ready
-    # API_BASE_URL_PRISON_PERSON: https://prison-person-api-dev.prison.service.justice.gov.uk
-    API_BASE_URL_PRISON_PERSON: https://prisoner-nomis-migration-dev.hmpps.service.justice.gov.uk
+    API_BASE_URL_PRISON_PERSON: https://prison-person-api-dev.prison.service.justice.gov.uk
```

Comment:
not to do with you, but odd duplication of `prison` - would have expected it to be https://person-api-dev.prison.service.justice.gov.uk 🤷